guides/common/modules/proc_creating-a-composite-content-view.adoc

```diff
@@ -5,17 +5,20 @@ Use this procedure to create a composite Content View.
 To use the CLI instead of the {ProjectWebUI}, see the xref:cli-creating-a-composite-content-view_{context}[].
 
 .Procedure
-. In the {ProjectWebUI}, navigate to *Content* > *Content Views* and click *Create New View*.
-. In the *Name* field, enter a name for the view.
+. In the {ProjectWebUI}, navigate to *Content* > *Content Views* and click *Create content view*.
+. In the *Create content view* window, enter a name for the view in the *Name* field.
 {ProjectName} automatically completes the *Label* field from the name you enter.
-. In the *Description* field, enter a description of the view.
-. Select the *Composite View?* checkbox to create a Composite Content View.
-. Optional: select the *Auto Publish* checkbox if you want the Composite Content View to be republished automatically when a Content View is republished.
-. Click *Save*.
-. In the *Add Content Views* area, select the Content Views that you want to add to the Composite Content View, and then click *Add Content Views*.
-. Click *Publish New Version* to publish the Composite Content View.
-In the *Description* field, enter a description and click *Save*.
-. Click *Promote* and select the lifecycle environments to promote the Composite Content View to, enter a description, and then click *Promote Version*.
+. Optional: In the *Description* field, enter a description of the view.
+. On the *Type* tab, select *Composite content view*.
+. Optional: If you want to automatically publish a new version of the Composite Content View when a Content View is republished, select the *Auto publish* checkbox.
+. Click *Create content view*.
+. On the *Content views* tab, select the Content Views that you want to add to the Composite Content View, and then click *Add content views*.
+. In the *Add content views* window, select the version of each Content View.
+. Optional: If you want to automatically update the Content View to the latest version, select the *Always update to latest version* checkbox.
+. Click *Add*, then click *Publish new version*.
+. Optional: In the *Description* field, enter a description of the Content View.
+. In the *Publish* pop-up window, set the *Promote* switch, then select the lifecycle environment.
```

Comment:
```suggestion
. In the *Publish* window, set the *Promote* switch, then select the lifecycle environment.
```
Since we decide to remove mentions to pop-up windows after all, please also remove this one. Once you've done that, this is good to go.